librz/arch/isa/msp430/msp430_il.c

```diff
@@ -0,0 +1,433 @@
+// SPDX-FileCopyrightText: 2024 Mostafa Mahmoud <ubermenchun@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <msp430/msp430_il.h>
+
+// ``The 16-bit program counter (PC/R0) points to the next instruction to be executed.``
```

Comment:
```suggestion
// The 16-bit program counter (PC/R0) points to the next instruction to be executed.
```

Also below. Or is this valid doxygen syntax I am not aware of?

Reply:
No, it's not valid doxygen. It's just  a quote from the manual.

Reply:
Please convert to Doxygent then.

Reply:
@moste00 Don't forget this one, please.